robyn/throttling.py

```diff
@@ -0,0 +1,28 @@
+import time
+from typing import Dict
+
+
+class RateLimiter:
+    def __init__(
+        self,
+        *,
+        calls_limit: int = 0,
+        limit_ttl: int = 0,
```

Comment:
One question - Is there a reason to allow only `kwargs`?

Reply:
I think having something like `RateLimiter(60, 120)` is very not intuitive and forces whoever reads the code to check it every time or to add some comment e.g. `# 60 calls per two minutes` - Forcing it to be `RateLimiter(calls_limit=60, limit_ttl=120)` is much more readable IMO.

Reply:
Wow! This makes sense. Perfect.